adi/fmcomms5.py

```diff
@@ -31,6 +31,7 @@
 # STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF
 # THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+import ad9361 as libad9361
```

Comment:
Will need to make pylibad9361 available to get CI to pass here. Likely should make this optional just in case users don’t have the library itself installed. Libad9361 shouldn’t be a hard requirement here